engine/crates/engine-v2/src/plan/attribution.rs

```diff
@@ -49,16 +52,16 @@ impl Attribution {
     }
 }
 
-#[derive(Debug)]
-pub struct ExtraField {
+#[derive(Debug, Clone)]
+pub struct ExtraField<SelectionSet = ExtraSelectionSetId> {
```

Comment:
`Clone` because a child plan needs to be cloneable for subscriptions which clones them at each new data input to continue the execution. Should be avoidable later. The generic is there because during planning I need a mapping from `FieldId` to fields to satisfy requirements but after this is an id.